tests/e2e/workout-management.spec.ts

```
@@ -305,42 +318,143 @@ test.describe('Workout Management', () => {
+      if (runnerCount < 2) {
+        console.log(
+          `Skipping multi-runner test: coach has ${runnerCount} connected runners (need at least 2)`
+        )
+        return
+      }
+
+      // Test with first runner
+      const firstRunnerCard = runnerCards.first()
+      const firstRunnerName =
+        (await firstRunnerCard.locator('[data-testid="runner-name"]').textContent()) || 'Runner 1'
+      const firstRunnerEmail =
```

ESLint

[error] 'firstRunnerEmail' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)